src/views/partials/newUserModal.ejs

```diff
@@ -1,17 +1,45 @@
-<form action='/newuser' method="post">
+<%- include('validation') -%> 
```

Comment:
huh I wonder, if we don't include the script tag in validation, can we include validation in the script tag where it is used?